### PR TITLE
feat: navigation system admin panel on top

### PR DESCRIPTION
**Pull request by bsilkyn**

### CODE REVIEW

File: frontend/src/components/layout/base/BaseHeader.vue [hunk 1]
```diff
@@ -1,31 +1,10 @@
 <script setup lang="ts">
```
Comment (tyboro2002) [inline frontend/src/components/layout/base/BaseHeader.vue:1]: why is all this code removed

**Review by tyboro2002 — CHANGES_REQUESTED** [review]

File: frontend/src/components/layout/base/BaseHeader.vue (hunk 1 above)
Comment (bsilkyn) [reply frontend/src/components/layout/base/BaseHeader.vue:1]: It hasn't been removed, it has been relocated.

File: backend/.flake8 [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [flake8]
 
 # Ignore unused imports
-ignore = F401
+ignore = F401,W504
```
Comment (tyboro2002) [inline backend/.flake8:4]: what is this doing here please add it also to the comment

File: frontend/tsconfig.json [hunk 3]
```diff
@@ -1,12 +1,13 @@
 {
     "compilerOptions": {
-        "target": "ES2020",
+        "target": "ESNext",
```
Comment (tyboro2002) [inline frontend/tsconfig.json:3]: what is the cause of this change

**Review by tyboro2002 — CHANGES_REQUESTED** [review]

File: backend/.flake8 (hunk 2 above)
Comment (bsilkyn) [reply backend/.flake8:4]: have added comments

File: frontend/tsconfig.json (hunk 3 above)
Comment (bsilkyn) [reply frontend/tsconfig.json]: I might have an idea what caused this, but that's the abstracted header file that has been removed

**Review by tyboro2002 — APPROVED** [review]

**Pull request merged**